storage/tianmu/handler/tianmu_handler.h

```diff
@@ -164,24 +164,24 @@ class TianmuHandler final : public handler {
   int fill_row(uchar *buf);
   int free_share();
 
-  std::shared_ptr<core::TableShare> share;
+  std::shared_ptr<core::TableShare> share_;
 
-  THR_LOCK_DATA m_lock; /* MySQL lock */
-  std::string m_table_name;
-  uint m_dupkey_pos = -1;
+  THR_LOCK_DATA lock_; /* MySQL lock */
+  std::string table_name_;
+  uint dupkey_pos_ = -1;
```

Comment:
assign negtive integer to an unsigned int may lead turn-flip. Becare about the symbol flag.